bootstrap-salt.sh

```diff
@@ -238,7 +238,11 @@ _ECHO_DEBUG=${BS_ECHO_DEBUG:-$BS_FALSE}
 _CONFIG_ONLY=$BS_FALSE
 _PIP_ALLOWED=${BS_PIP_ALLOWED:-$BS_FALSE}
 _PIP_ALL=${BS_PIP_ALL:-$BS_FALSE}
-_SALT_ETC_DIR=${BS_SALT_ETC_DIR:-/etc/salt}
+if uname -a | grep FreeBSD > /dev/null; then
+  _SALT_ETC_DIR=${BS_SALT_ETC_DIR:-/usr/local/etc/salt}
```

Comment:
There are different indentation style of 2 spaces instead of 4 in all lines you have changed.